lib/wings.rb

```diff
@@ -8,7 +8,19 @@ module Wings
 
 require 'valkyrie'
 require 'wings/resource_factory'
+require 'wings/metadata_adapter'
```

Comment:
And this can be replaced along with the updates are made to `wings.rb` on `master`.